pages/common/msmtp.md

```diff
@@ -0,0 +1,16 @@
+# msmtp
+
+> An SMTP client.
+> In the default sendmail mode, msmtp reads a mail from standard input and sendscit to an SMTP server for delivery.
+
+- Send a mail using the default account configured in `~/.msmtprc`:
+
+`echo {{"Hello world"}} | msmtp {{to@example.org}}`
+
+- Send a mail using a specific account configured in `~/.msmtprc`:
+
+`echo {{"Hello world"}} | msmtp --account={{example}} {{to@example.org}}`
+
+- Send a mail without a configured account:
+
+`echo {{"Hello world"}} | msmtp --host={{localhost}} --port={{999}} --from={{from@example.org}} {{to@example.org}}`
```

Comment:
I am not entirely with this but as far as I know the "sent on behalf of" is shown when an optional head `Sender:` is set. So if i would send an email on someone else's behalf I would put my email address in `Sender:`.

But none of this has anything to do with this. The `--from` argument sets the content of the `From:` header and `--host` specifies the host the smtp server is running, nothing special about it.